src/analysis.jl

```diff
@@ -130,3 +130,10 @@ function potential_energy(s::Simulation)
 
     return uconvert(s.energy_unit, potential)
 end
+
+function vacf(vl::VelocityLogger)
```

Comment:
Maybe the name `velocityautocorr` is clearer? Also, export the function and add a docstring.